readme.md

```diff
@@ -326,7 +326,7 @@ Thanks for contributing! 🦋🙌
 - [](# "mark-private-orgs") [Marks private organizations on your own profile.](https://user-images.githubusercontent.com/6775216/44633467-d5dcc900-a959-11e8-9116-e6b0ffef66af.png)
 - [](# "profile-hotkey") Adds a keyboard shortcut to visit your own profile: <kbd>g</kbd> <kbd>m</kbd>.
 - [](# "show-user-top-repositories") [Adds a link to the user’s most starred repositories.](https://user-images.githubusercontent.com/1402241/48474026-43e3ae80-e82c-11e8-93de-159ad4c6f283.png)
-- [](# "hide-user-forks") [Hides forks and archived repos from profiles (but they can still be shown).](https://github-production-user-asset-6210df.s3.amazonaws.com/140871606/257037977-5707d20e-73b6-4908-9d22-0f6a127aabaa.png)
+- [](# "hide-user-forks") [Hides forks and archived repos from profiles](https://github-production-user-asset-6210df.s3.amazonaws.com/83146190/262547716-e75e5641-25a6-4af7-b60c-4e4488e2b3ef.png) [(but they can still be shown.)](https://github-production-user-asset-6210df.s3.amazonaws.com/83146190/262547829-2da96718-fb18-4f2e-b637-8157f552e278.png)
```

Comment:
Unfortunately this feels like I'm playing "find the differences" between the two screenshots. Let's add some text, you can use this URL:

![Screenshot 1](https://github.com/refined-github/refined-github/assets/1402241/85cf0951-c6ed-45fe-8cfc-e447e3ed2a25)